bitsandbytes/functional.py

```diff
@@ -1472,23 +1566,22 @@ def gemv_4bit(
     out: Tensor = None,
     transposed_A=False,
     transposed_B=False,
-    state=None
```

Comment:
While quant_state is preferable, this would break code that used the different keyword. As such, to ensure backwards compatibility with the API, its better to leave the name as `state`. One could deprecate this, but I think its not worth it and we should just leave it as `state`

Reply:
done.
My idea was to name references to `MatmulLtState` class and to `QuantState` class differently.
Now reverted to the original signature in both `gemv_4bit()` (where it expects QuantState) and in `mm_dequant()` (where I was wrong - it was just some old format of quant_state).